stackprinter/frame_formatting.py

```diff
@@ -180,91 +173,87 @@ def _format_listing(self, lines, lineno):
         msg += self.sep_source_below
         return msg
 
-    def _format_assignments(self, assignments, truncation=500):
+    def _format_assignments(self, assignments):
         msgs = []
         for name, value in assignments.items():
             val_str = format_value(value,
                                    indent=len(name) + self.var_indent + 3,
-                                   truncation=truncation)
+                                   truncation=self.truncate_vals)
             assign_str = self.val_tpl % (name, val_str)
             msgs.append(assign_str)
         if len(msgs) > 0:
             return self.sep_vars + '\n' + ''.join(msgs) + self.sep_vars + '\n\n'
         else:
             return ''
 
-
-def select_scope(fi, lines, lines_after, show_vals, show_signature,
```

Comment:
This has been converted to a method so the whole thing is indented. The only changes are whitespace and `self`s. Use the "Hide whitespace changes" feature of GitHub.